staging/dex-k8s-authenticator/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "v1.1.0"
+description: "Authenticator for using Dex with Kubernetes"
+name: dex-k8s-authenticator
+version: 1.1.1
+sources:
+- https://github.com/mintel/dex-k8s-authenticator
+maintainers:
```

Comment:
Are they the maintainers ? @mhrabovcin Please, add yourself here :)

Reply:
Done @hectorj2f